agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataCollector.java

```diff
@@ -79,11 +82,13 @@ private static CpuPerformanceCounterCalculator getCpuPerformanceCounterCalculato
       throw td;
     } catch (Throwable t) {
       try {
-        LoggerFactory.getLogger(QuickPulseDataCollector.class)
-            .error(
-                "Could not initialize {}",
-                CpuPerformanceCounterCalculator.class.getSimpleName(),
-                t);
+        try (MDC.MDCCloseable ignored = QUICK_PULSE_SEND_ERROR.makeActive()) {
```

Comment:
I don't think the above block can throw an exception (anymore, I think it used to at one point)

Reply:
i found 40808 records though.

Reply:
can you send me the log and stack trace from one of these in teams?

Reply:
removed try/catch block.